doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,132 @@
+# User Guide `exasollog`
```

Comment:
I think you should add a developers_guilde, that includes information on how to run the unit tests

Reply:
Definitely. Wrote ticket #4.